internal/pkg/cli/flag.go

```diff
@@ -230,6 +231,7 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	githubURLFlagDescription         = "(Deprecated.) Use '--url' instead. Repository URL to trigger your pipeline."
 	githubAccessTokenFlagDescription = "GitHub personal access token for your repository."
 	gitBranchFlagDescription         = "Branch used to trigger your pipeline."
+	connectionFlagDescription        = "CodeStar Connections connection to your third-party repository."
```

Comment:
I assume the input is a name right instead of an ARN? should we update the text?
```suggestion
	connectionFlagDescription        = "Optional. CodeStar connection name to connect to your repository."
```